composer.json

```diff
@@ -65,8 +65,7 @@
     "monolog/monolog": "1.23.*",
     "newerton/jquery-mousewheel": "dev-master",
     "pamelafox/lscache": "1.0.5",
-    "malihu/malihu-custom-scrollbar-plugin": "3.1.5",
-    "grimmlink/qtip2": "3.0.3"
```

Comment:
Since this is a CSS only solution, we are only removing a JS dependency from `composer.json`.